static/templates/compose_announce.handlebars

```diff
@@ -0,0 +1,8 @@
+<div class="compose-all-everyone">
```

Comment:
I feel we should also rename the class name to `compose-all-everyone-announce`, but no strong opinion here.

Reply:
OK, I renamed it in 181a2cd.

Reply:
OK, I changed it in 181a2cd.